ios/FluentUI/Core/Colors.swift

```diff
@@ -257,6 +257,35 @@ public final class Colors: NSObject {
             }
         }
 
+        public var colorSet: ColorSet {
+            switch self {
+            // Set of colors defined as global tokens
+            case .communicationBlue:
```

Comment:
Sorry if I'm not reading something right, but why are we returning the same token for each of these color cases? Is there also a reason we're explicitly returning the same value for each case, rather than falling through to a single return value?

Reply:
Because I made a dum mistake and didn't commit the right version of this file 😨 pls hold

Reply:
Fixed! Thanks for the catch :)